### DISCUSSION TITLE
Inventory System in Budibase

### DISCUSSION THREAD

**Discussion by rmutili** [Budibase/budibase#2082] [Ideas]
Hi @mjashanks,

About last discussion on inventory management, have you thought about how to initiate a reorder level based on a given number within the products? Also when requesting products from warehouse, we would like to select multiple products from a table and submit for a approval, once approved, the products are deducted from the specific warehouse.

**Comment by mjashanks** [comment]
Hi @rmutili 

I logged this idea after talking to you https://github.com/Budibase/budibase/discussions/2024, which is what we will need to implement your reorder level requirement. We will be prioritising some items this week, so will find out where this can fit into the roadmap. 

To Select multiple items for approval, you could setup a Many-to-Many relationship between a `Products` table and a `Requests` table. This means that you could structure your request form as below, where you have a multi-select for `Products`. 

I believe that you may also want to filter or search your products dropdown. You cannot do this at the moment, but we are going to add this feature this month as part some broader work on forms: https://github.com/Budibase/budibase/discussions/2033 (we might not get all of the things listed here done this month).

![image](https://user-images.githubusercontent.com/3524181/126966140-b30f4a66-9a71-4bf6-bd18-34c845368f34.png)

**Reply by rmutili** [reply]
Sounds good, let me know how it goes...

**Reply by rmutili** [reply]
Wanted to add some image based on what to expect...

![Web Table](https://user-images.githubusercontent.com/21999305/127327771-c0de4161-1709-4f47-94c9-7aaab5bb1b23.PNG)
